camera/src/main/java/com/kylecorry/andromeda/camera/Camera.kt

```diff
@@ -112,6 +121,7 @@ class Camera(
             return
         }
 
+        cameraProvider?.unbindAll()
```

Comment:
Once the change is made in flipCamera per my other comment, this should not be needed anymore